### DISCUSSION TITLE
Is virtiofs supported?

### DISCUSSION THREAD

**Discussion by daxiguafan** [smartxworks/virtink#105] [Q&A] [answered]
Is virtiofs supported? or has the plan?

**Comment by fengye87** [comment, marked as answer]
Yes, it's supported. Though we haven't add any docs about it, it's easy to guess its usage from [the code](https://github.com/smartxworks/virtink/blob/main/pkg/apis/virt/v1alpha1/types.go#L59). Cheers!

**Reply by daxiguafan** [reply]
thks
